### DISCUSSION TITLE
Additional Text Field for Discord Show Status

### DISCUSSION THREAD

**Discussion by llama8** [s0hv/foo_discord_rich#19] [Ideas]
Hi!

Your fork has been working like a charm. I wanted to ask if there is any chance to have a new optional Text Field or Line for Discord Status, so for eg, I can have Top Text Field for song %title%, Bottom Text Field for %Album Artist%, and finally a new optional line or Text Field for %Album%.

This is how I have it configured right now with Top Text Field and Bottom Text Field only.
https://ibb.co/jRGgXPk

Unless there's a way to add a new line from those Text Fields, would be cool to have a third optional Text Field for showing more info

My best regards to you

**Comment by s0hv** [comment]
Discord only allows two lines of customizable text so what you're looking for is not possible in discord. If you want %Album% to show up when certain conditions are met, you can use title formatting for that.

**Reply by Azuriye** [reply]
If I recollect, I remember someone talking about how they would rewrite a Music Player's RPC entirely to use the Game SDK Rich Presence to send data as that allows more fields? that would include a whole rewrite though and I'm not sure if that's another way some rich presence clients accomplish it.
